.travis/fedora/travis-tasks.sh

```diff
@@ -10,12 +10,51 @@ RETRY_CMD=/builddir/.travis/retry-command.sh
 
 pushd /builddir/
 
+valgrind_cmd='
+    valgrind --error-exitcode=1
+             --errors-for-leak-kinds=definite
+             --leak-check=full
+             --show-leak-kinds=definite
+             --suppressions=/usr/share/glib-2.0/valgrind/glib.supp
+             --suppressions=/builddir/contrib/valgrind/libmodulemd-python.supp'
+
 # Build the code under GCC and run standard tests
 meson --buildtype=debug \
       $COMMON_MESON_ARGS \
       travis
 
-meson test -C travis --num-processes=$PROCESSORS --print-errorlogs -t 5
+meson test --suite smoketest \
+           -C travis \
+           --num-processes=$PROCESSORS \
```

Comment:
Should `$COMMON_MESON_ARGS` also be included for each of the `meson test ...` runs? Although, the way things were re-worked by this PR, the dirty test flag set by `$COMMON_MESON_ARGS` is superfluous everywhere except for the `dirty` suite below--in which case `-Dtest_dirty_git=true` could be hard coded since the shell conditional has already determined if it should be run or not.

Reply:
No, the meson args are only for setting up the buildroot. The `meson test` calls wouldn't know what to do with those flags.

I hard-coded the `-Dtest_dirty_git=true` in the meson buildroot command.